app/controllers/api_controller/parser/request_adapter.rb

```diff
@@ -15,6 +15,10 @@ def api_prefix
         @api_prefix ||= "#{base}#{prefix}"
       end
 
+      def attributes
+        @attributes ||= @params['attributes'].blank? ? [] : @params['attributes'].split(',')
```

Comment:
Why the ternary operator vs original `@attributes ||= params['attributes'].to_s.split(",")`?